tests/core/test_langchain_llm.py

```
@@ -0,0 +1,152 @@
+"""Tests for LangChainLLMAdapter: invoke, stream, retry, message conversion."""
+
+from __future__ import annotations
+
+import asyncio
```

Flake8 (7.3.0)

[error] 'asyncio' imported but unused

(F401)